src/app/public/modules/flyout/fixtures/flyout-fixtures.module.ts

```diff
@@ -51,11 +51,11 @@ import {
   ],
   imports: [
     CommonModule,
-    NoopAnimationsModule,
     RouterTestingModule,
     SkyFlyoutModule,
     SkyModalModule,
-    SkyToastModule
+    SkyToastModule,
+    NoopAnimationsModule
```

Comment:
?

Reply:
I realized here that the test classes were also importing the `NoopAnimationsModule` that was needed because the order of imports matter. The Other modules were importing the actual animation module which was turnning back on the animations that had been disabled by the `NoopAnimationsModule` above. Moving the `NoopAnimationsModule` to the bottom ensures it is imported last and actually does what it is intended to do.